snapcraft/internal/pluginhandler/__init__.py

```diff
@@ -24,12 +24,13 @@
 import subprocess
 import sys
 from glob import glob, iglob
-from typing import cast, Dict, List, Set, Sequence
+from typing import Any, cast, Dict, List, Set, Sequence  # noqa: F401
 
 import snapcraft.extractors
 from snapcraft import file_utils, yaml_utils
 from snapcraft.internal import common, elf, errors, repo, sources, states, steps
 from snapcraft.internal.mangling import clear_execstack
+from snapcraft.project import Project
```

Comment:
only import this if TYPE_CHECKING

Reply:
what's the rationale for that? that goes back to that other thread I forget which...

Reply:
(I assume it was a dependency issue, but I didn't see any resulting problems?)

Reply:
Done.